agents/che-core-api-agent/src/test/resources/run_launcher_bats_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
As a general tendency, if something is postponed then it won't be done at all :(
`docker-maven-plugin` allows starting container in privileged mode and then we can execute tests script inside it.
May be I am mistaken but could you clarify what issue you faced here?